code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1460,6 +1467,7 @@ var/list/robot_verbs_default = list(
 	scrambledcodes = 1
 	req_access = list(access_cent_specops)
 	ionpulse = 1
+	ert_upgrade = 1

```

Comment:
This is about equivelent to `if(istype(src, /mob/living/silicon/robot/ert))`, and at least that makes it clear what you're doing.

Rather than snowflaking the ERT borgs all around the code, why not add more booleans in general to configure this kind of stuff? For example, you could replace l273 with
```
/mob/living/silicon/robot
    var/list/force_modules = null

l273
if(islist(force_modules) && force_modules.length)
    modules = force_modules.Copy()
```
and etc for other parts of this, such as `var/can_rename`